code/modules/surgery/organs/organ_internal.dm

```diff
@@ -290,6 +304,8 @@
 	say_mod = "buzzes"
 
 /obj/item/organ/internal/tongue/fly/TongueSpeech(var/message)
+	spawn(0)
+		speakDevilName(message)
```

Comment:
Yes, neither I nor Remie could see a better way of implementing this.  However, I am of the belief that while somewhat snowflakey, this is a feature that will add a lot to the antag.

I will look for a better way to implement this, but tongue seemed like a very good place, as I would need the message pre speech impediment, and also to prevent unneeded checks for when xenos/drones/sillicons speak.